controllers/controllers.articles.js

```diff
@@ -23,3 +24,17 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+
+  selectCommentsByArticleId(article_id)
+    .then((rows) => {
```

Comment:
here is where you can make use of `two` model functions, including your `selectArticleById` to check that the `article_id` is in the database